webpack.config.js

```diff
@@ -0,0 +1,57 @@
+var path = require('path');
+var webpack = require('webpack');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    library: 'AFRAME',
+    libraryTarget: 'umd',
+    path: path.resolve(__dirname, 'dist'),
+    publicPath: '/dist/',
+    filename: 'aframe-master.js'
+  },
+  devtool: 'source-map',
+  mode: 'development',
+  devServer: {
+    host: 'local-ip',
+    port: process.env.PORT || 9000,
+    hot: false,
+    liveReload: true,
+    static: {
+      directory: 'examples'
+    }
```

Comment:
For you to understand, before you had a specific build for budo and replaced /dist by /build in the script. In webpack-dev-server, the build is done in memory when you are accessing `/dist/aframe-master.js`, all the other files are served from the examples directory.